src/IotHub/IotHub/IotHub/AddAzureRmIotHubCertificate.cs

```diff
@@ -144,9 +144,11 @@ public override void ExecuteCmdlet()
 
                     CertificateBodyDescription certificateBodyDescription = new CertificateBodyDescription();
                     certificateBodyDescription.Certificate = certificate;
+                    CertificateProperties certificateProperties = new CertificateProperties();
+                    certificateProperties.Certificate = certificateBodyDescription.Certificate;
 
                     CertificateDescription certificateDescription;
-                    certificateDescription = this.IotHubClient.Certificates.CreateOrUpdate(this.ResourceGroupName, this.Name, this.CertificateName, this.Etag, certificateBodyDescription.Certificate);
+                    certificateDescription = this.IotHubClient.Certificates.CreateOrUpdate(this.ResourceGroupName, this.Name, this.CertificateName, this.Etag, certificateProperties);
```

Comment:
do we want to point to a copy rather than the original object here?

Reply:
As we discussed offline, since we are adding certificate, there is no original object, instead we would need to create a new object in order to use CreateOrUpdate. Also tested that only pass in the certificate is enough, the rest CertificateProperties properties can be figured out after the service call, this is the return after calling this command:

Id                : /subscriptions/a386d5ea-ea90-441a-8263-d816368c84a1/resourceGroups/jiacjutest/providers/Microsoft.D
                    evices/IotHubs/newstate/certificates/certifitry
ResourceGroupName : jiacjutest
Name              : newstate
CertificateName   : certifitry
Subject           : management
Thumbprint        : 46BB6DAA9A62A4D256FC02A1F73AAB18F7724B72
Status            : Unverified
Expiry            : 9/27/2031 12:09
Created           : 1/01/0001 00:01
Etag              : IjhlMDI2ODQ1LTAwMDAtMDMwMC0wMDAwLTYzZWQ4YzZjMDAwMCI=